### ISSUE TITLE
Coding Retreat Ideas

### ISSUE THREAD

**Issue by linamnt** [UofTCoders/council#261]
This issue is for brainstorming ideas for a fall coding retreat for the exec (and other members?). 
Things to consider:
- How much would we personally be willing to pay?
- How much would UofT Coders cover (I feel this should only be the case if open to all members)
- What are we comfortable with (staying overnight vs day trip? being in a cabin in the wilderness? do we have enough drivers if it's out of the city? etc.)

Suggestions so far:
- Hart House Farms Trip (there's a house where you can sleep over 50ppl, with a nice kitchen and living space)
    - $25 pp for overnight stay, or $15 pp for a day trip
    - description: _Among the maple sugar bush and cleared fields, a well-trodden path will take you on a two-hundred-foot descent through a unique limestone ecology to the palisade._    
    - no alcohol allowed on property as of last year.
- Airbnb somewhere outside Toronto, maybe somewhere with options for hikes or something outdoorsy, or other activities?

**Comment by QuLogic** [comment]
Another question is when? I would guess that unless we're going to a different UofT campus, then the weekend (Fri night to Sun night)? And if that's the case, then when in the year? If weather is a concern, then maybe October (which is actually very soon!)?

**Comment by linamnt** [comment]
Agreed it is very soon, but if we can all agree to a weekend in October (13-14, 20-21 work for me...) then we can just try to find a place? November isn't that bad either...

**Comment by SaraMati** [comment]
Two places I've heard of, put here to look into: 

- Wolf  Den Hostel, just a few kilometres west of Algonquin Park and adjacent to the Oxtongue River
- Geneva  Park Lodge, Orilia

**Comment by QuLogic** [comment]
For Wolf Den: the lodge rooms are $43-$75/night, max 2-3 people; bunkhouses are $246/night, max 4 people; independent cabins mostly fit 4, but the one that fits 7 is $291/night (min 2 nights) on the night I checked. (note: other than the cabin, those are single person bookings; there's a slight discount for multiple people in a room) Prices may or may not double on high demand nights.

There's a parkbus to Algonquin, but the schedule seems to end on October 8 (and it's very sparse anyway). Trip is anywhere from 2.5-4 hours depending on where you get picked up and it's $89 return.

Generally, weekends in October are already booked, weekdays are not too bad. November is also not particularly booked (including weekends.)

**Comment by linamnt** [comment]
I've heard [Retreatify](https://www.retreatify.com/) is good for planning small (to large) retreats. If we're down for maybe an end of spring, early summer retreat to work on the Reproducibility project, we can start planning now a time that would be good. I think early June is best for me
